app/src/main/kotlin/com/wire/android/ui/settings/devices/DeviceDetailsViewModel.kt

```diff
@@ -119,7 +119,7 @@ class DeviceDetailsViewModel @Inject constructor(
                     isE2eiCertificateActivated = true,
                     e2eiCertificate = certificate.certificate,
                     isLoadingCertificate = false,
-                    device = state.device.updateE2EICertificateStatus(certificate.certificate.status)
+                    device = state.device.updateE2EICertificate(certificate.certificate)
```

Comment:
Question: can it be covered by tests, I see we have a testing class for this viewmodel :D

Reply:
done